package.json

```diff
@@ -52,6 +52,7 @@
     "flow-bin": "^0.74.0",
     "jest-junit": "^3.7.0",
     "react-test-renderer": "^15.5.0",
+    "redux-mock-store": "^1.5.3",
```

Comment:
Since we're not using this anymore, please remove it and run `npm prune`. commit the package/package.lock file after that.